<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tagesansicht</class>
 <widget class="QDialog" name="Tagesansicht">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tagesansicht</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="neuer_Termin">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>920</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Termin hinzufügen</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tag_vor">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(123, 186, 225);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="tag_zurueck">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(123, 186, 225);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Date">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Date Label&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>631</width>
     <height>661</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>200</y>
     <width>631</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>150</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Das steht heute an:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Lernplan">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>920</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lernplan </string>
   </property>
  </widget>
  <widget class="QPushButton" name="faecher">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>910</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Studienfächer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profil">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>920</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Profil</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ansicht">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Tagesansicht</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wochenansicht</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monatsansicht</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Aktuallisieren</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hallo User! Heute ist der </string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(6, 6, 6);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>neuer_Termin</zorder>
  <zorder>tag_vor</zorder>
  <zorder>tag_zurueck</zorder>
  <zorder>Date</zorder>
  <zorder>calendar</zorder>
  <zorder>tableView</zorder>
  <zorder>tableView_2</zorder>
  <zorder>label_2</zorder>
  <zorder>Lernplan</zorder>
  <zorder>faecher</zorder>
  <zorder>profil</zorder>
  <zorder>ansicht</zorder>
  <zorder>pushButton</zorder>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
